gdk/CLIParser.py

```diff
@@ -180,6 +181,8 @@ def _add_common_args_for_all_commands(self):
 
 def main():
     try:
+        Emit().installed_metric()
```

Comment:
why "installed"? this just fires every time you do anything at all right because this is in main.

Reply:
I mentioned it on the readme. I know this would fire every time. Next PR will address that. Right now it is disabled. This is similar to what SAM_CLI does they do it on their main and record that they emitted it and after that they stop.